src/akamai/js/akamaiHD_flash.js

```diff
@@ -15,7 +15,9 @@ require("../../../html5-common/js/utils/constants.js");
    * Config variables for paths to flash resources.
    */
 
-  var pluginPath;
+  var scriptsList = document.currentScript;
+  var akamaiPath = scriptsList.src;
+  var pluginPath = akamaiPath.slice(0, 1 + akamaiPath.lastIndexOf("/"));
```

Comment:
@stephanieblotner Looks like this is going to mean that we must ask customers not to change the filename of the akamaiHD_flash video plugin.  When this plugin is completed we will need to document this